hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -251,11 +251,20 @@
           name: "HI_SPEED_SEL",
           mubi: true,
           desc: '''
-            A value of kMultiBitBool4True selects nominal speed external clocks.
+            A value of kMultiBitBool4True selects nominal speed external clock.
             All other values selects low speed clocks.
 
             Note this field only has an effect when the !!EXTCLK_CTRL.SEL field is set to
             kMultiBitBool4True.
+
+            Nominal speed means the external clock is approximately the same frequency as
+            the internal oscillator source.  When this option is used, all clocks operate
+            at roughly the nominal frequency.
+
+            Low speed means the external clock is apporximately half the frequency as the
```

Comment:
Nit: "approximately". Also, this should probably be "... half the frequency **of** the ..."

Reply:
sounds good.